x-pack/plugins/stack_alerts/server/alert_types/index_threshold/alert_type.ts

```diff
@@ -180,7 +180,7 @@ export function getAlertType(
         groupResult.metrics && groupResult.metrics.length > 0 ? groupResult.metrics[0] : null;
       const value = metric && metric.length === 2 ? metric[1] : null;
 
-      if (!value) {
+      if (value === null) {
```

Comment:
nit: should we also check for `value === undefined` just to cover all our bases?

Reply:
It also could be a shorter version:
`if (value == null) {`
but current way probably is more clear